armi/reactor/__init__.py

```diff
@@ -51,7 +51,7 @@
 See :doc:`/developer/index`.
 """
 
-from typing import Dict, Callable, Union, TYPE_CHECKING
+from typing import TYPE_CHECKING, Callable, Dict, Union
```

Comment:
I have no strong opinions on the sorting of imports. Should we standardize the practice across armi (e.g., a ruff plugin?) so that we don't have a lot of these unrelated changes in some PRs?

Could be a useful addition to our dev guide if there are vs code settings that will do this automatically too

cc @john-science @opotowsky @albeanth

Reply:
Yeah VSCode is automatically sorting imports alphabetically for me, I haven't figured out how to turn it off yet. According to VSCode, it's doing this according to `ruff` rules, but I don't see it happening for anybody else so I must be doing something wrong.

Reply:
Okay I was able to turn it off, so no more rampant import sorting